Process Layout

Anjela-Mayer opened the pull request:
Added Pathtool to VRProcessLayout to draw Process Subjects and Messages. 
Extended VRProcess by functions to access Process Messages.

Victor-Haefner commented:
Es gibt ein paar sachen die ich nicht optimal finde
- cscope.out, was ist das für eine Datei?
- in src/addons/Semantics/Reasoning/VROWLImport.cpp
   - Zeile 415/416, das return muss da raus, oder?
   - Die neue Funktion VROWLImport::printOWLSubjects() ist hier fehl am Platz, bitte zu VRProcess verschieben, OWL ist ein Format für Ontologien, die haben nicht zwanksläufig etwas mit Prozessen zu tun.

Anjela-Mayer commented:
- removed cscope.out 
- removed return statement  (line 415/416 src/addons/Semantics/Reasoning/VROWLImport.cpp )
- removed printOWLSubjects() testfunction from src/addons/Semantics/Reasoning/VROWLImport.cpp
